.github/CODEOWNERS

```diff
@@ -43,17 +43,14 @@
 # PRLabel: %Batch
 /sdk/batch/                                         @gingi @paterasMSFT @dpwatrous
 
-# PRLabel: %Communication
-/sdk/communication/                                 @JianpingChen @ankitarorabit @minnieliu @Azure/azure-sdk-communication-code-reviewers
```

Comment:
don't we want to keep it as discussed in the teams chat? (or are we going to add it back alter?)

Reply:
I removed it as per comment from @DominikMe:
"Is there a value in keeping the top-value sdk/communicationpath? CognitiveServices doesn't have a base path only the sub-paths. But maybe the base path is needed to map as "Communication" labeled issues to an assignee? It would be better anyway to have issues be directed towards individual services with a "Communication - <service>" label and avoid one more hop."